llvm/test/DebugInfo/Generic/mixed-source.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: %llc_dwarf -O0 -filetype=obj -o - < %s | llvm-dwarfdump -debug-line - | FileCheck %s
+
+; CHECK: include_directories[  0] = "dir"
+; CHECK-NEXT: file_names[  0]:
+; CHECK-NEXT:            name: "foo.c"
+; CHECK-NEXT:       dir_index: 0
+; CHECK-NEXT:          source: "void foo() { }\n"
+; CHECK-NEXT: file_names[  1]:
+; CHECK-NEXT:            name: "bar.h"
+; CHECK-NEXT:       dir_index: 0
+; CHECK-NOT:           source:
+
+; Ensure we reject debug info where the DIFiles of a DICompileUnit mix source
```

Comment:
I believe this is a copy paste mistake from the test borrowed from the reverted commit?